services/movie_session.py

```diff
@@ -0,0 +1,57 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
+    movie_session = MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
+    )
+    return movie_session
+
+
+def get_movies_sessions(
+        session_date: datetime = None)\
+        -> QuerySet[MovieSession]:
+    movie_session = MovieSession.objects.all()
+    if session_date:
+        movie_session = MovieSession.objects.filter(
+            show_time__date=session_date
+        )
+
+    return movie_session
+
+
+def get_movie_session_by_id(
+        movie_session_id: int
+) -> QuerySet[MovieSession]:
+
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(session_id: int,
+                         show_time: datetime = None,
+                         movie_id: int = None,
+                         cinema_hall_id: int = None) -> None:
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]